recipe/multitenancy/multitenancyClaims.go

```diff
@@ -0,0 +1,84 @@
+package multitenancy
+
+import (
+	"errors"
+	"time"
+
+	"github.com/supertokens/supertokens-golang/recipe/multitenancy/multitenancyclaims"
+	"github.com/supertokens/supertokens-golang/recipe/session/claims"
+	"github.com/supertokens/supertokens-golang/supertokens"
+)
+
+func NewMultitenancyClaims() (*claims.TypeSessionClaim, *claims.TypeSessionClaim, multitenancyclaims.TypeMultitenancyClaimValidators) {
+	fetchTenantId := func(userId string, userContext supertokens.UserContext) (interface{}, error) {
+		instance, err := GetRecipeInstanceOrThrowError()
+		if err != nil {
+			return nil, err
+		}
+		tenantIdRes, err := instance.GetTenantIdForUserID(userId, userContext)
+		if err != nil {
+			return false, err
+		}
+		if tenantIdRes.OK != nil {
+			return tenantIdRes.OK.TenantId, nil
+		} else {
+			return "", errors.New("UNKNOWN_USER_ID")
+		}
+	}
+
+	fetchDomains := func(userId string, userContext supertokens.UserContext) (interface{}, error) {
+		instance, err := GetRecipeInstanceOrThrowError()
+		if err != nil {
+			return nil, err
+		}
+		tenantIdRes, err := instance.GetTenantIdForUserID(userId, userContext)
+		if err != nil {
+			return false, err
+		}
+		if tenantIdRes.OK != nil {
+			tenantId := tenantIdRes.OK.TenantId
+			domains, err := instance.GetDomainsForTenantId(tenantId, userContext)
```

Comment:
this function may be nil. In this case, you do not want to add this claim value at all.

Reply:
Added a check